Source/AtlusScriptLibrary/FlowScriptLanguage/Compiler/FlowScriptCompiler.cs

```diff
@@ -125,6 +125,37 @@ public void AddListener(LogListener listener)
         listener.Subscribe(mLogger);
     }
 
+    /// <summary>
+    /// Tries to get a list of all files that would be imported (directly or transitively) when compiling a flowscript file.
+    /// </summary>
+    /// <param name="files">A List of paths to .bf, .flow, and .msg files to be used as a base when checking for imports.</param>
+    /// <param name="resolvedImports">A list of all imports found. This includes the passed in <paramref name="files"/>.</param>
+    /// <returns>True if imports could be resolved, false otherwise</returns>
+    public bool TryGetImports(List<string> files, out string[] resolvedImports)
+    {
+        CompilationUnit compilationUnit = new CompilationUnit();
```

Comment:
Would it be a lot of work to refactor to get rid of the dummy compilation unit (using just an array of Imports)? 
Would be a bit cleaner that way.

Reply:
I'm not sure how hard it'd be but I'll give it a try. 
This was definitely me being a bit lazy and just wanting something that worked quickly 😅

Reply:
Any updates?

Reply:
Sorry not yet. I did a bit when you initially asked and realised it's going to be more work than I first thought. Nothing I can't do but I think it'll take a couple of hours which I just haven't found yet. 

I'll try and do it this weekend so we can close this off.